apmpackage/apm/0.1.0/data_stream/app_metrics/elasticsearch/ilm/default_policy.json

```diff
@@ -0,0 +1,27 @@
+{
+    "policy": {
+        "phases": {
+            "warm": {
+                "min_age": "30d",
+                "actions": {
+                    "readonly": {},
+                    "set_priority": {
+                        "priority": 50
+                    }
+                }
+            },
+            "hot": {
+                "min_age": "0ms",
```

Comment:
Is the min_age needed on the hot phase? I don't think it's normally needed when defining a policy in ES.

Reply:
good catch - it is not, if the `min_age` is not configured it defaults to `0`.